Ion.egg-info/SOURCES.txt

```diff
@@ -875,7 +875,6 @@ intranet/middleware/url_slashes.py
 intranet/settings/.gitignore
 intranet/settings/__init__.py
 intranet/settings/ci_secret.py
-intranet/settings/secret.py
```

Comment:
A rollback to this change was requested in the original PR. However, it seems to me that this change is both necessary and proper, since `secret.py` is not in source control and doesn't need to be part of the egg. Further testing may be necessary.